platforms/core-runtime/files/src/main/java/org/gradle/internal/file/nio/PosixJdk7FilePermissionHandler.java

```diff
@@ -14,20 +14,21 @@
  * limitations under the License.
  */
 
-package org.gradle.internal.nativeintegration.filesystem.jdk7;
+package org.gradle.internal.file.nio;
 
-import org.gradle.internal.nativeintegration.filesystem.FileModeAccessor;
-import org.gradle.internal.nativeintegration.filesystem.FileModeMutator;
+import org.gradle.internal.file.FileModeAccessor;
+import org.gradle.internal.file.FileModeMutator;
 
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.attribute.PosixFileAttributeView;
 import java.nio.file.attribute.PosixFileAttributes;
 
-import static org.gradle.internal.nativeintegration.filesystem.jdk7.PosixFilePermissionConverter.convertToInt;
-import static org.gradle.internal.nativeintegration.filesystem.jdk7.PosixFilePermissionConverter.convertToPermissionsSet;
+import static org.gradle.internal.file.nio.PosixFilePermissionConverter.convertToInt;
+import static org.gradle.internal.file.nio.PosixFilePermissionConverter.convertToPermissionsSet;
 
+@SuppressWarnings("Since15")
```

Comment:
❓ Also here, is it safe to suppress "Since15"?

Should we add a comment why it is safe?